.vscode-template/settings.json

```diff
@@ -9,8 +9,6 @@
     "**/*.class": true,
     "**/*.hasTasty": true,
     "**/target/": true,
-    "community-build/community-projects": true,
-    "scala2-library/{doc,docs,lib,META-INF,scripts,spec,test,tools}/": true, // only allow scala-backend/src
-    "scala2-library/src/[abcefimprs]*": true
+    "community-build/community-projects": true
```

Comment:
I would add "tests" to this since getting the tests in the search (including the standard lib newly added there) is never what we want.